Sources/TableViewDriver.swift

```diff
@@ -35,7 +35,7 @@ open class TableViewDriver: NSObject {
     }
 
     /// The table view to which the `TableViewModel` is rendered.
-    public let tableView: UITableView
+    let tableView: TableView
```

Comment:
This seemed okay to do?

Reply:
I haven't tried it internally, but I think in most cases, you'll have another pointer to this view anyway.